SECURITY.md

```diff
@@ -1,6 +1,75 @@
-# Reporting Security Issues
+# Security Policy
 
-If you believe you have found a security vulnerability in PyTorch, we encourage you to let us know right away. We will investigate all legitimate reports and do our best to quickly fix the problem.
+ - [**Using Pytorch Securely**](#using-pytorch-securely)
+   - [Untrusted models](#untrusted-models)
+   - [Untrusted inputs](#untrusted-inputs)
+   - [Data privacy](#data-privacy)
+   - [Untrusted environments or networks](#untrusted-environments-or-networks)
+   - [Multi-Tenant environments](#multi-tenant-environments)
+ - [**Reporting a Vulnerability**](#reporting-a-vulnerability)
+
+## Using Pytorch Securely
+### Untrusted models
+Be careful when running untrusted models. This classification includes models created by unknown developers or utilizing data obtained from unknown sources.
```

Comment:
>  or utilizing data obtained from unknown sources.
Can you please share a link on some research on dangers of using data from unknown sources?

Reply:
Sure! I think the main threat here is data poisoning, and you can find nice discussions about it in the following links:
1. https://arxiv.org/abs/2312.04748 
2. https://arxiv.org/abs/2401.05566
3. https://blog.mithrilsecurity.io/poisongpt-how-we-hid-a-lobotomized-llm-on-hugging-face-to-spread-fake-news/
4. https://elie.net/blog/ai/attacks-against-machine-learning-an-overview/#data-poisoning

Reply:
I think it's worth sharing the two papers here to make it clear.

Reply:
I've added them as footnotes, PTAL

Reply:
Looks good!